binding/dotnet/README.md

```diff
@@ -27,6 +27,10 @@ Rhino is:
 
 ## Compatibility
 
+Platform compatible with .NET 6.0+:
```

Comment:
.NET or .NET core? I don't know actually, just asking

Reply:
`.NET` :) They keep changing the name